WNPRC_Purchasing/src/org/labkey/wnprc_purchasing/WNPRC_PurchasingManager.java

```diff
@@ -333,10 +333,9 @@ else if (null != lineItem.get("quantityReceived"))
         }
         catch (Exception e)
         {
-            List<ValidationException> rowErrors = new ArrayList<>();
-            rowErrors.add(new ValidationException().addError(new SimpleValidationError(e.getMessage())));
-            return rowErrors;
+            throw new Exception(e.getMessage(), e);// is there a less generic exception to use to capture exceptions from insertRows, updateRows, and deleteRows in addition to other exceptions (like IllegalStateException)?
```

Comment:
I don't see any value in this catch block. If you delete it completely, the original exception will propagate up the call stack and you don't need to create a new exception

Reply:
yes indeed, was wandering if there was a better exception to throw